src/styles/header.scss

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(https://fonts.google.com/specimen/Roboto);
```

Comment:
The `src` property in the `@font-face` rule should point to a valid font file URL, not a Google Fonts page URL. Consider using a direct link to the font file.